TerminalDocs/index.md

```diff
@@ -11,50 +11,37 @@ ms.service: terminal
 
 # What is Windows Terminal?
 
-The Windows Terminal is a new, modern, fast, efficient, powerful, and productive terminal application for users of command-line tools and shells like Command Prompt, PowerShell, and WSL. Its main features include multiple tabs, Unicode and UTF-8 character support, a GPU accelerated text rendering engine, and custom themes, styles, and configurations.
+The Windows Terminal is a new, modern, fast, efficient, powerful, and productive terminal application for users of command line tools and shells like Command Prompt, PowerShell, and Windows Subsystem for Linux (WSL). Its main features include multiple tabs, panes, Unicode and UTF-8 character support, a GPU accelerated text rendering engine, and custom schemes and configurations.
 
-![WindowsTerminal screenshot](./images/terminal-command-args.gif)
+![Windows Terminal screenshot](./images/overview.png)
 
 > [!NOTE]
 > [What's the difference between a console, a terminal, and a shell?](https://www.hanselman.com/blog/WhatsTheDifferenceBetweenAConsoleATerminalAndAShell.aspx) Read Scott Hanselman's explanation.
```

Comment:
_THIS_. Can we do this formatting anytime there's a paragraph that says "NOTE". We should just do this more throughout the docs overall